packages/cli/cloud/src/create-project/action.ts

```diff
@@ -0,0 +1,61 @@
+import inquirer from 'inquirer';
+import { AxiosError } from 'axios';
+import { defaults } from 'lodash/fp';
+import type { CLIContext, ProjectAnswers, ProjectInput } from '../types';
+import { tokenServiceFactory, cloudApiFactory, local } from '../services';
+
+function handleError(ctx: CLIContext, error: Error) {
+  const tokenService = tokenServiceFactory(ctx);
+  const { logger } = ctx;
+
+  logger.debug(JSON.stringify(error));
+  if (error instanceof AxiosError) {
+    switch (error.response?.status) {
+      case 401:
+        logger.error('Your session has expired. Please log in again.');
+        tokenService.eraseToken();
+        return;
+      case 403:
+        logger.error(
+          error.response.data ||
+            'You do not have permission to create a project. Please contact support for assistance.'
+        );
+        return;
+      case 400:
+        logger.error('Invalid input. Please check your inputs and try again.');
```

Comment:
I think we can improve this error message, it's really vague. Do you not return a good error message from the API you can utilise?

Reply:
I believe we can improve as well, would it make sense to put some effort on this side for a next iteration?